cmd/neofs-adm/internal/modules/morph/n3client.go

```diff
@@ -20,12 +20,16 @@ type clientContext struct {
 }
 
 func getN3Client(v *viper.Viper) (*client.Client, error) {
+	// number of opened connections
+	// by neo-go client per one host
+	const maxConnsPerHost = 10
+
 	ctx := context.Background() // FIXME(@fyrchik): timeout context
 	endpoint := v.GetString(endpointFlag)
 	if endpoint == "" {
 		return nil, errors.New("missing endpoint")
 	}
-	c, err := client.New(ctx, endpoint, client.Options{})
+	c, err := client.New(ctx, endpoint, client.Options{MaxConnsPerHost: maxConnsPerHost})
```

Comment:
Actually, `neofs-adm` might ignore this limit.

The limit is important for running applications that reuse neo-go connection over time. `neofs-adm` runs in a single thread and exits on the finish, so it can live with default 0. I don't mind to set it though, do it as you wish.

Reply:
thought about it, but decided that to have a limit and do not use it is better than not to have